modules/core/shared/src/main/scala/lucuma/core/model/sequence/flamingos2/Flamingos2DynamicConfig.scala

```diff
@@ -6,29 +6,32 @@ package lucuma.core.model.sequence.flamingos2
 import cats.Eq
 import cats.syntax.all.*
 import lucuma.core.enums.*
+import lucuma.core.math.Wavelength
 import lucuma.core.util.TimeSpan
 import monocle.Focus
 import monocle.Lens
 
 case class Flamingos2DynamicConfig(
-  exposure:       TimeSpan,
-  disperser:      Option[Flamingos2Disperser],
-  filter:         Flamingos2Filter,
-  readMode:       Flamingos2ReadMode,
-  lyot:           Flamingos2LyotWheel,
-  fpu:            Flamingos2FpuMask,
-  readoutMode:    Option[Flamingos2ReadoutMode], // engineering
-  reads:          Option[Flamingos2Reads], // engineering
-) {
-  val isImaging: Boolean       = fpu.isImaging
-  val isMOS: Boolean           = fpu.isMOS
-  val isLongSlit: Boolean      = fpu.isLongSlit
-  val decker: Option[Flamingos2Decker] = fpu.fold(none, _.value.decker.some, _ => none)
-}
+  exposure:    TimeSpan,
+  disperser:   Option[Flamingos2Disperser],
+  filter:      Flamingos2Filter,
+  readMode:    Flamingos2ReadMode,
+  lyotWheel:   Flamingos2LyotWheel,
+  fpu:         Flamingos2FpuMask,
+  decker:      Flamingos2Decker,
```

Comment:
`decker` appears in the F2 long slit configuration as an option, so if it is selected to be anything other than the default we'll need to keep up with it.